nifi-nar-bundles/nifi-soap-bundle/nifi-soap-processors/src/main/java/org/apache/nifi/processors/soap/GetSOAP.java

```diff
@@ -214,106 +276,178 @@ public void onScheduled(final ProcessContext context) {
         options.setCallTransportCleanup(true);
```

Comment:
May want to move logic in OnScheduled to OnTrigger, especially the ServiceClient as that is not thread safe. The properties may also change based on the incoming FF and would thus need to be accessed in the OnTrigger